sockets provider build warnings

@jithinjosepkl 

prov/sockets/src/sock_cq.c: In function \u2018sock_cq_sreadfrom\u2019:
prov/sockets/src/sock_cq.c:252: warning: \u2018end_ms\u2019 may be used uninitialized in this function
  CC     prov/sockets/src/src_libfabric_la-sock_cntr.lo
prov/sockets/src/sock_cntr.c: In function \u2018sock_cntr_wait\u2019:
prov/sockets/src/sock_cntr.c:136: warning: \u2018end_ms\u2019 may be used uninitialized in this function
  CC     prov/sockets/src/src_libfabric_la-sock_poll.lo
  CC     prov/sockets/src/src_libfabric_la-sock_wait.lo
  CC     prov/sockets/src/src_libfabric_la-sock_ep_rdm.lo
  CC     prov/sockets/src/src_libfabric_la-sock_ep_dgram.lo
  CC     prov/sockets/src/src_libfabric_la-sock_ep_msg.lo
prov/sockets/src/sock_ep_msg.c: In function \u2018sock_pep_listener_thread\u2019:
prov/sockets/src/sock_ep_msg.c:1047: warning: \u2018entry_sz\u2019 may be used uninitialized in this function

--- a/prov/sockets/src/sock_cntr.c
+++ b/prov/sockets/src/sock_cntr.c
@@ -133,7 +133,7 @@ static int sock_cntr_set(struct fid_cntr *cntr, uint64_t value)
 static int sock_cntr_wait(struct fid_cntr *cntr, uint64_t threshold, int timeout)
 {
 	int ret = 0;
-	double start_ms, end_ms;
+	double start_ms = 0, end_ms = 0;
 	struct sock_cntr *_cntr;
 	
 	_cntr = container_of(cntr, struct sock_cntr, cntr_fid);

--- a/prov/sockets/src/sock_cq.c
+++ b/prov/sockets/src/sock_cq.c
@@ -249,7 +249,7 @@ ssize_t sock_cq_sreadfrom(struct fid_cq *cq, void *buf, size_t count,
 	int ret = 0;
 	int64_t threshold;
 	struct sock_cq *sock_cq;
-	double start_ms, end_ms;
+	double start_ms = 0, end_ms = 0;
 	ssize_t cq_entry_len, avail;
 	
 	sock_cq = container_of(cq, struct sock_cq, cq_fid);

--- a/prov/sockets/src/sock_ep_msg.c
+++ b/prov/sockets/src/sock_ep_msg.c
@@ -1119,6 +1119,7 @@ static void *sock_pep_listener_thread (void *data)
 			SOCK_LOG_INFO("Received SOCK_CONN_SHUTDOWN\n");
 			conn_response = (struct sock_conn_response*)conn_req;
 
+			entry_sz = sizeof(*cm_entry);
 			cm_entry->fid = conn_response->hdr.c_fid;
 			fid_ep = container_of(conn_response->hdr.c_fid, 
 					      struct fid_ep, fid);

